controller/setting_controller.go

```diff
@@ -996,3 +1000,36 @@ func (sc *SettingController) updateInstanceManagerCPURequest() error {
 
 	return nil
 }
+
+func (sc *SettingController) cleanupFailedSupportBundles() error {
+	failedLimit, err := sc.ds.GetSettingAsInt(types.SettingNameSupportBundleFailedHistoryLimit)
+	if err != nil {
+		return err
+	}
+
+	if failedLimit > 0 {
+		return nil
+	}
```

Comment:
This is for the cleanup. The setting controller will remove all failed support bundles if the limit is set to 0.

Reply:
oh, I was confused. Because I thought when failedLimit == 0, it means we don't have any limitation. Then, we need to have an explanation for the setting.